triton-cli/test-vectors/simple.tasm

```diff
@@ -0,0 +1,1740 @@
+push 10
```

Comment:
A file with 1740 lines doesn't exactly scream “simple” to me… :astonished: Would it make sense to use something smaller for testing?

Reply:
Yea it's a short program that i duplicated with a vim macro. I removed all but 25 lines.